test/e2e/frontend/cypress/tests-ee/team/library.spec.js

```diff
@@ -44,9 +44,12 @@ describe('FlowForge - Library', () => {
         it('allows users to create new Blueprints if they don\'t have  any', () => {
             interceptBlueprints()
 
+            cy.get('[data-el="ff-tab"]').contains('Blueprints').click()
```

Comment:
Because we changed the redirect, each time we access the team/library pages, we're redirected to the shared library page instead of the blueprints page. We have to take this into account and 'manually' navigate to the blueprints page thus making all subsequent checks valid again. The same goes for the following scenarios in this group.